src/PKSim.Core/Services/TransportContainerUpdater.cs

```diff
@@ -58,9 +59,27 @@ public void SetDefaultSettingsForTransporter(ISimulationSubject simulationSubjec
          foreach (var transporterContainer in simulationSubject.AllMoleculeContainersFor<TransporterExpressionContainer>(transporter))
          {
             updateTransporterContainerFromTemplate(transporterContainer, null, transportType);
+            updateFractionExpressedEpithelial(transporterContainer, transportType);
          }
       }
 
+      private void updateFractionExpressedEpithelial(TransporterExpressionContainer transporterContainer, TransportType transportType)
+      {
+         if (!transportType.IsOneOf(TransportType.Efflux, TransportType.Influx, TransportType.PgpLike))
+            return;
+         
+         var fractionExpressedEpithelial = transporterContainer.Parameter(CoreConstants.Parameters.FRACTION_EXPRESSED_EPITHELIAL);
+         if (fractionExpressedEpithelial == null)
+            return;
+
+         //value was set by the user.
+         if (fractionExpressedEpithelial.Value != 0 && fractionExpressedEpithelial.Value != 1)
+            return;
+         
+         //Set the value according to the new transport type (Efflux one, all other 0)
+         fractionExpressedEpithelial.Value = transportType == TransportType.Efflux ? 1 : 0;
```

Comment:
```
         //Set the value according to the new transport type (Efflux one, all other 0)
         fractionExpressedEpithelial.Value = transportType == TransportType.Efflux ? 1 : 0;
```
I think it's not correct for PgP. I would expect that fraction_expressed_endothelial = 1 for PgP (same as for Efflux)

Reply:
oh sorry I read same as Influx. Will correct in next PR